README.md

```diff
@@ -1,12 +1,34 @@
-# How to write exercises for the learn-ocaml platform? (Resources)
+#  Step 2: Basic grading by comparison with your solution
 
 This repository contains the source files for the tutorial
 
 [How to write exercises for the learn-ocaml platform?](https://github.com/ocaml-sf/learn-ocaml/blob/master/docs/howto-write-exercises.md)
 
-To get the source files for step X do:
+To go to another step and get the corresponding source files do (for step X):
 ```
 git checkout step-X
 ```
 
-For instance, for step 1, do `git checkout step-1`.
\ No newline at end of file
+## What is to learn in this step?
+
+How to:
+
+* Grade an arity-one function by comparison to a given solution.
+
+* Grade multiple-arguments functions.
+
+* Grade a polymorphic function with different input types.
+
+## List of graded functions
+
+### grade-function/test.ml
+
+* `identity`: [`identity x`] returns x. Testing is done on integer inputs.
```

Comment:
The style is very telegraphic. Are you describing the tests? If this is the case, say it explicitely.